components/TagSimple.js

```diff
@@ -0,0 +1,8 @@
+
+export default function TagSimple({ name }) {
+  return (
+  <div className="text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1" >
+     {name}
+  </div>
+  );
+}
```

Comment:
Now we have 2 tag components, should we use a `tag` folder and have them both in there?